doc/library/tensor/basic.txt

```diff
@@ -676,6 +676,20 @@ Creating Tensor
 
     Returns a tensor filled with 1s that has same shape as `x`.
 
+.. function:: zeros(shape, dtype=None)
+
+    :param shape: a tuple/list of scalar with the shape information.
+    :param dtype: the dtype of the new tensor. If None, will use floatX.
+
+    Returns a tensor filled with 0s of the provided shape.
+
+.. function:: ones(shape, dtype=None)
+
+    :param shape: a tuple/list of scalar with the shape information.
+    :param dtype: the dtype of the new tensor. If None, will use floatX.
+
+    Returns a tensor filled with 1s of the provided shape.
+
```

Comment:
I have a tendency to add small stuff in PR. Probably not the best, but as travis is slow for our big test suite, it help us.